bins/synthesis.cpp

```diff
@@ -21,20 +21,7 @@ using namespace spot;
 static SynthesisMeasure* g_synt_measure = nullptr;
 static SynthesisCLIOptions options;
 
-void on_sighup(int args) {
-    try {
-        dump_measures(*g_synt_measure, options);
-    } catch (const std::runtime_error& re) {
-        std::cout << "Runtime error: " << re.what() << std::endl;
-    } catch (const std::exception& ex) {
-        std::cout << "Error occurred: " << ex.what() << std::endl;
-    } catch (...) {
-        std::cout << "Unknown failure occurred. Possible memory corruption"
-                  << std::endl;
-    }
-
-    exit(EXIT_SUCCESS);
-}
+void on_sighup(int args);
```

Comment:
looks weird to me.
why do you need so many files?
I suggest the following refactor - remove all the files in this project, create a `index.js` with the following code:

```
function isReactiveSynthesis(bool) {
   return bool
}
```

Let me know if you have any questions.